packages/console-client/src/test/client/index.test.ts

```diff
@@ -447,4 +447,21 @@ describe('console-client', () => {
     const { calls } = axiosMock.mock
     assert.equal(calls.length, 3)
   })
+
+  it('exposes userinfo fluent APIs correctly', async(t) => {
```

Comment:
Let's avoid testing specific API for the moment, we'd be better to reason some way to automate these instead.

Otherwise if every new API is to tested on its own we have no benefits on automating client generation

Reply:
Removed